webapp/maps_heatmaps/.firebase/hosting.cHVibGlj.cache

```diff
@@ -0,0 +1,4 @@
+app.js.LICENSE.txt,1600088034138,016f5c4839abd92d0f29f5ba08954088af4a0cbcb4e56f1ded08c1b41574d96b
+style.css,1600074492110,d1dc5d6eece0e46eeeb074e3caab9570c56fa836f5e44d1d3cf18de09253ab6e
+index.html,1600163465516,99236b482cf86f5c06a46f167ab5d6ab3544cc4f5081dd1fa42606af8234aef2
+app.js,1600163678617,5f4baeb598da6882aaf9eafc1a3d08b889698a2a0702ec7d2da20242c0941b90
```

Comment:
What is this file? I think it should not be committed, right?
1. Add it to .gitignore here: https://github.com/googleinterns/ellipsis-step-internship/blob/master/webapp/maps_heatmaps/.gitignore
2. When making a PR always check the changes that it results in, using:
https://github.com/googleinterns/ellipsis-step-internship/pull/14/files
and verify that everything that is there should be there

Reply:
file added to gitignored and removed from commit.
We will check the changes next time.